hack/download-charts.sh

```diff
@@ -72,6 +72,12 @@ function patchIstioCharts() {
   resources: ["securitycontextconstraints"] \
   resourceNames: ["privileged"] \
   verbs: ["use"]/' "${CHARTS_DIR}/cni/templates/clusterrole.yaml"
+
+  # Remove fsGroup, runAsUser, runAsGroup from istiod deployment so that it can run on OpenShift
```

Comment:
Is there any risk these get reintroduced in Istio and we never notice it because of this?

Reply:
Definitely possible, but I think we can remove this patch as soon as we stop packaging 1.19 and earlier (i.e. as soon as 1.21.0 is released, so that we have at least two different minor versions in the operator).

Reply:
ok, perhaps we should add a `#FIXME` comment so we don't forget...